pom.xml

```diff
@@ -9,12 +9,11 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+        <maven.compiler.release>21</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <junit.jupiter.version>5.9.2</junit.jupiter.version>
-        <assertj.core.version>3.23.1</assertj.core.version>
-        <mockito.version>5.2.0</mockito.version>
+        <junit.jupiter.version>5.10.2</junit.jupiter.version>
+        <assertj.core.version>3.25.2</assertj.core.version>
```

Comment:
The latest version is [3.25.3](https://mvnrepository.com/artifact/org.assertj/assertj-core)